app/scripts/metamask-controller.js

```diff
@@ -1565,7 +1565,8 @@ export default class MetamaskController extends EventEmitter {
         },
       },
       env: {
-        isAccountSyncingEnabled: !isProduction() && isManifestV3,
+        isAccountSyncingEnabled:
+          (!isProduction() && isManifestV3) || process.env.IN_TEST,
```

Comment:
This is a bit convoluted, but as we don't want to release on Firefox just yet, we still want our E2Es to be played on Firefox.

Reply:
NVM, reverted this lol.